collect/metric_objects.go

```diff
@@ -0,0 +1,50 @@
+package collect
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/numaproj-labs/numaflow-perfman/util"
+)
+
+const (
+	minutes = int(util.RateInterval / time.Minute)
+	seconds = int((util.RateInterval % time.Minute) / time.Second)
+)
+
+type MetricObject struct {
```

Comment:
Do you we see other commands apart from collect using these metrics objects and defined metrics as well? If yes might be better to have a separate package for metrics so that all operations and definitions can be decoupled from collect.

Reply:
currently only collect command uses them, but i think this is a good idea to keep things modular for the future. Will create separate metrics package